app/controllers/state_file/questions/retirement_income_controller.rb

```diff
@@ -29,6 +22,19 @@ def update
 
       private
 
+      def load_1099_r
+        @state_file1099_r = current_intake.state_file1099_rs.find(params[:id])
+      end
+
+      def load_warnings
+        if @state_file1099_r.state_tax_withheld_amount == 0 || @state_file1099_r.state_tax_withheld_amount.nil?
+          @state_file1099_r.errors.add(:state_tax_withheld_amount, I18n.t("state_file.questions.retirement_income.edit.state_tax_withheld_absent_warning"))
+        end
+        if @state_file1099_r.state_tax_withheld_amount.present? && @state_file1099_r.state_tax_withheld_amount > @state_file1099_r.gross_distribution_amount
+          @state_file1099_r.errors.add(:state_tax_withheld_amount, I18n.t("activerecord.errors.models.state_file1099_r.errors.must_be_less_than_gross_distribution", gross_distribution_amount: @state_file1099_r.gross_distribution_amount))
+        end
+      end
+
```

Comment:
just recapping for my self here -- these warnings are non-blocking potentially (they don't factor into the `update` action, which validates using `:retirement_income_intake` context